### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stiyyagura0901**
changes reflective orb name change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+# evals-orb
+# evals orb [![CircleCI Build Status](https://circleci.com/gh/CircleCI-Public/evals-orb.svg?style=shield "CircleCI Build Status")](https://circleci.com/gh/CircleCI-Public/evals-orb) [![CircleCI Orb Version](https://badges.circleci.com/orbs/circleci/evals.svg)](https://circleci.com/orbs/registry/orb/circleci/evals) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/circleci-public/evals-orb/main/LICENSE) [![CircleCI Community](https://img.shields.io/badge/community-CircleCI%20Discuss-343434.svg)](https://discuss.circleci.com/c/ecosystem/orbs)
 
-This repository has the code for the the CircleCI [AI Evals Orb](https://github.com/CircleCI-Public/ai-evals-orb). Please note that this version is currently in beta. Explore, understand, and experiment, but proceed with caution when integrating it into practical applications. 🛠✨
+This repository has the code for the the CircleCI [Evals Orb](https://github.com/CircleCI-Public/evals-orb).
 
 ## Usage
 
 ### Setup
```
Comment (BeFunes) [inline README.md:9]: Is there something we should say here about the Github token necessary for PRs to be posted on GitHub?

Comment (stiyyagura0901) [reply README.md:9]: done.

File: src/examples/run_langsmith_eval.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 description: >
-  Run LLM evaluations with langsmith using the ai-evals orb
+  Run LLM evaluations with langsmith using the evals orb
```
Comment (BeFunes) [inline src/examples/run_langsmith_eval.yml]: LangSmith

File: src/examples/run_braintrust_eval.yml [hunk 3]
```diff
@@ -1,9 +1,9 @@
 description: >
-  Run LLM evaluations with braintrust using the ai-evals orb
+  Run LLM evaluations with braintrust using the evals orb
```
Comment (BeFunes) [inline src/examples/run_braintrust_eval.yml]: Braintrust with capital B

File: README.md [hunk 4]
```diff
@@ -1,26 +1,26 @@
...
 
-In order to use the AI Evals Orb on CircleCI you will need to setup environment variables either in context or project env vars. Required variables are BRAINTRUST_API_KEY.
+In order to use the Evals Orb on CircleCI checkout [llm-eval-examples](https://github.com/CircleCI-Public/llm-eval-examples) for usage examples. Email ai-feedback@circleci.com for feedback.
```
Comment (BeFunes) [inline README.md]: I think let's keep what you had before if it's still the case. 
You can just add "For usage examples, refer to [llm-eval-examples](https://github.com/CircleCI-Public/llm-eval-examples)"

Comment (stiyyagura0901) [reply README.md]: done. added some more stuff.

File: src/examples/run_langsmith_eval.yml (hunk 2 above)
Comment (stiyyagura0901) [reply src/examples/run_langsmith_eval.yml]: done.

File: src/examples/run_braintrust_eval.yml (hunk 3 above)
Comment (stiyyagura0901) [reply src/examples/run_braintrust_eval.yml]: done.

**Review by ryan-circleci — APPROVED** [review]

**Pull request merged**